### PR TITLE
8755 - Fixed bug where chart is cut off on bottom legend placement mode

### LINKED ISSUE #8755: Donut: Chart cut off
The donut is behaving inconsistently, it is being cut off in some cases and cannot see legends that are placed in bottom for banner widgets, 
Here is the stackblitz link for the reproduced issue, 
[Donuts cut off](https://stackblitz.com/edit/ids-quick-start-1771-y2n5tj?file=src%2Fapp%2Fapp.component.ts,src%2Fapp%2Fapp.component.html)

Issues found:

- Donut is cutting in middle in some cases, few of those cases are while opening and closing inspect tab,
[screen-capture (5).webm](https://github.com/infor-design/enterprise/assets/167510255/c7d21ba2-d007-4969-9e28-619421f60ca6)
- The donut should be displayed in whole and we should also see legends that are not being shown initially

We are using ids version: 4.92.2 and beta SDK 8.
We can also see another issue that busy indicator can only be seen at the end of the completion of loading this behavior is only seen in donuts with bottom legend placements.

Please note that these issues are only found in donuts placed in banner

For full details refer to teams post:
[Teams Post](https://teams.microsoft.com/l/message/19:2b0c9ce520b0481a9ce115f0ca4a326f@thread.skype/1716367241073?tenantId=457d5685-0467-4d05-b23b-8f817adda47c&groupId=4f50ef7d-e88d-4ccb-98ca-65f26e57fe35&parentMessageId=1716367241073&teamName=IDS%20Enterprise%20Development&channelName=General&createdTime=1716367241073)

### PR DESCRIPTION
**Pull request by yohannahbautista**
**Explain the _details_ for making this change. What existing problem does the pull request solve?**

Adjusted height so that chart won't be cut off

Note: In the stackblitz example, the widget doesn't have auto-height. That also helps a bit in adjusting the height of the chart.

**Related github/jira issue (required)**:

Closes #8755 

**Steps necessary to review your pull request (required)**:

- Pull this branch, build and run the app
- Go to: http://localhost:4000/components/donut/test-legend-bottom.html
- Chart should not be cut off
- Resize browser to smaller widths
- Chart should not be cut off

**Included in this Pull Request**:
- [ ] A note to the change log.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -25,6 +25,7 @@
 - `[Datagrid]` Fixed an error that occurred in some situations. ([#8709](https://github.com/infor-design/enterprise/issues/8709))
 - `[Datagrid]` Fixed a bug where date would be cleared after updating cell on different locales. ([NG#1710](https://github.com/infor-design/enterprise-ng/issues/1710))
 - `[Datepicker]` Fixed focus date not properly assigned when calendar is used as datepicker. ([#8585](https://github.com/infor-design/enterprise/issues/8585))
+- `[Donut]` Fixed bug where chart is cut off on bottom legend placement mode. ([#8755](https://github.com/infor-design/enterprise/issues/8755))
```
Comment (tmcconechy) [inline docs/CHANGELOG.md:28]: Is this in the 4.97 section? Should be starting a new section now.

**Comment by tmcconechy** [comment]
Also:

a) Can we remove the lines (trying not to promote that option)
b) Doesnt look totally centered
c) The legend is not centered when in desktop mode

Screenshot 2024-06-06 at 10 07 13 AM
Screenshot 2024-06-06 at 10 07 05 AM

**Comment by ericangeles** [comment]
My only concern here is by using `auto-width` class. I believe the content developers don't have any control in the widget (parent) section. They can only access the `widget-content` as far as I remember.

**Comment by ericangeles** [comment]
I am also thinking of putting it on the homepage example to make sure it will really work there.

**Comment by ericangeles** [comment]
@yohannahbautista, we can discuss it offline if it's confusing to you.

**Comment by yohannahbautista** [comment]
> My only concern here is by using `auto-width` class. I believe the content developers don't have any control in the widget (parent) section. They can only access the `widget-content` as far as I remember.

@ericangeles It's not auto-width, it's auto-height. The auto-height class gives a min-height that helps keep the pie chart from being cut off, I just added additional style to include the extra height needed when the legends are at the bottom

**Comment by ericangeles** [comment]
Yeah, I thought I said `auto-height`. Our minimum height for widget/card is 360px if I'm not mistaken. 

cc: @tmcconechy

**Comment by ericangeles** [comment]
`auto-height` class doesn't have a minimum height. Its height is based on the contents of the widget. In this case, if the widget does have a lot of data (e.g., legends), it will push the widget.

**Comment by tmcconechy** [comment]
@yohannahbautista @ericangeles 360 is the height for a small widget. But we have half size widgets and charts could be any size you want within reason. The chart should render based on whatever size parents it is in.

**Review by ericangeles — DISMISSED** [review]

**Review by jbrcna — APPROVED** [review]
image
image
image
image

**Review by n-ace-ancog — APPROVED** [review]
Working on my end.

image

image

**Review by glenlieorillo — APPROVED** [review]
image

image

**Pull request merged**